data/tools/incident_response.json

```
@@ -62,6 +62,16 @@
       "keywords": "HybridAnalysis, Any.Run, URLhaus, MISP, CAPE, Malshare, Valhalla, Hashlookup, malware",
       "blackarch": "munin-hashchecker"
     },
+    {
+      "name": "osquery",
+      "website": "https://osquery.io/",
+      "source": "https://github.com/osquery/osquery",
+      "description": "Endpoint visibility tool providing SQL-based querying for monitoring and security analysis of operating systems.",
+      "language": "C++",
```

```suggestion
      "language": "Cplusplus",
```

> The language field value should match [Font Mfizz](http://fizzed.com/oss/font-mfizz) Programming languages name when it exists and be capitalized
>
> Example: Cplusplus for C++ or Nodejs for Node.js

https://inventory.raw.pm/docs/docs/contribution_guidelines/format.html